nuxeo-core/nuxeo-core-event/src/main/java/org/nuxeo/ecm/core/scheduler/SchedulerServiceImpl.java

```diff
@@ -154,20 +151,8 @@ public void start(ComponentContext context) {
 
     protected void startScheduler() {
         ClusterService clusterService = Framework.getService(ClusterService.class);
-        Duration duration;
-        try {
-            String prop = Framework.getProperty(CLUSTER_START_DURATION_PROP);
-            if (isBlank(prop)) {
-                duration = Duration.ZERO;
-            } else {
-                duration = DurationUtils.parse(prop);
-            }
-        } catch (DateTimeParseException e) {
-            duration = Duration.ZERO;
-        }
-        if (duration.isZero() || duration.isNegative()) {
-            duration = Duration.ofMinutes(1);
-        }
+        String prop = Framework.getProperty(CLUSTER_START_DURATION_PROP);
+        Duration duration = DurationUtils.parsePositive(prop, Duration.ofMinutes(1));
```

Comment:
It might be more clear to have 1 as a constant: since there is no default for `org.nuxeo.scheduler.cluster.start.duration`, most of the time it is this default parameter of `DurationUtils#parsePositive` that will be used.

Though it's the caller's responsibility to set the timeout, as we are always using 1 minute, it could be in DurationUtils:
```
public static final long DURATION_DEFAULT_1_M = 1;
```

Reply:
Ok for a constant but I don't want to push it to `DurationUtils` itself, I think it's too low level to have any responsibility in the default value.